public-images/public-images-how-to/launch-with-libvirt.rst

```diff
@@ -0,0 +1,147 @@
+.. _launch-libvirt:
+
+Launch QCOW images using libvirt
+================================
+
+Amongst the virtualization tools available for Ubuntu, there is `libvirt`_. `libvirt`_ is not a hypervisor, but rather a set of tools and libraries to manage various other hypervisors, such as `KVM`_ and `QEMU`_.
+
+
+Although there are many more `applications using libvirt`_, this guide will focus on common ones. Those can be installed with of the following command:
```

Comment:
There's a typo in the second sentence here, and it could stand to be reworded by bringing the content of the sentence after the code block up here.